src/appsignal/config.py

```diff
@@ -90,6 +87,7 @@ def set_private_environ(config: Options):
         "_APPSIGNAL_APP_ENV": config.get("environment"),
         "_APPSIGNAL_APP_NAME": config.get("name"),
         "_APPSIGNAL_HOSTNAME": config.get("hostname"),
+        "_APPSIGNAL_LANGUAGE_INTEGRATION_VERSION": f"python-{__version__}",
```

Comment:
Consider moving it to the `CONSTANT_PRIVATE_ENVIRON` instead, as it's not derived from the config?